### ISSUE TITLE
Loading presets in Aeolus fails

### ISSUE THREAD

**Issue by riban-bw** [zynthian/zynthian-issue-tracking#1078]
# Describe the issue
It has been reported that loading presets in Aeolus does not work. I am adding this ticket so that it gets investigated but at this time I have no further info other than @jofemodo says it may be related to [this commit](https://github.com/zynthian/zynthian-ui/commit/864b9de8a1b231206c1dcb9c9adc956be1d3b708).
# To Reproduce
## Steps to reproduce the behavior:
1. Load a preset in Aeolus
 ## Expected behaviour
Preset is loaded.
## Actual behaviour
Preset is not loaded.

# Configuration
## Hardware
```
Raspberry Pi 4 Model B Rev 1.5
Audio: V5 ADAC
Display: MIPI DSI 800x480 (inverted)
Wiring: V5
I2C: MCP23017@0x20, MCP23017@0x21
Profile: v5
```
## System
```
Debian GNU/Linux 12 (bookworm)
Build Date: 2024-02-01
Memory: 13% (496M/3792M)
SD Card: 88% (24G/29G)
Temperature: 44.8ºC
Overclock: Maximum
```
## MIDI & UI
```
Tuning: 440 Hz
Master Channel: 16
Preload Presets: Off
ZS3 (SubSnapShots): On
Power Save: 10 minutes
Audio Levels on Snapshots: On
```
## Software
```
zyncoder: oram (a7976d8) 
zynthian-ui: oram (b53cbe6)
zynthian-sys: oram (b8541bc)
zynthian-data: oram (db0355a)
zynthian-webconf: oram (b098993)
```

**Comment by riban-bw** [comment]
This may be related to the change to fix presets. Presets now use a dictionary for the divisions (previously a list) which fixes abiltiy to select individual manual presets. The default presets in `/zynthian/zynthian-data/presets/aeolus.json` were not updated at this time. (I must have forgotten to push the changes.) This is now fixed by commit 9958411cf72e6d64d57d5f7b3380541eedc3d826.

What do we do about existing users with 3 broken default presets? Can we ask them to recreate them and save them? Do we auto-detect the broken preset format? Maybe the latter isn't too bad.

**Comment by riban-bw** [comment]
Now fixed for legacy snapshots.
